changelog/debug-nothrow.dd

```diff
@@ -0,0 +1,14 @@
+Nothrow code can now be used in debug blocks
+
+When writing debug code, one isn't interested in the type safety, but a pleasant
```

Comment:
I don’t think `nothrow` has any thing to do with type safety. This is clearly a copy paste from the other PR for `@safe`.